src/main/kotlin/no/nav/familie/tilbake/behandling/BehandlingService.kt

```diff
@@ -441,6 +446,21 @@ class BehandlingService(
             BehandlingMapper.tilDomeneBehandling(opprettTilbakekrevingRequest, fagsystem, fagsak, ansvarligsaksbehandler)
         behandlingRepository.insert(behandling)
 
+        val manuelleBrevmottakere = brevmottakere.map { brevmottaker ->
+            ManuellBrevmottaker(
+                behandlingId = behandling.id,
+                type = MottakerType.valueOf(brevmottaker.type.name),
+                adresselinje1 = brevmottaker.manuellAdresseInfo.adresselinje1,
+                adresselinje2 = brevmottaker.manuellAdresseInfo.adresselinje2,
+                postnummer = brevmottaker.manuellAdresseInfo.postnummer,
+                poststed = brevmottaker.manuellAdresseInfo.poststed,
+                landkode = brevmottaker.manuellAdresseInfo.landkode,
+                navn = brevmottaker.navn,
+                vergetype = brevmottaker.vergetype
+            )
+        }
+        manuellBrevmottakerRepository.insertAll(manuelleBrevmottakere.toList())
```

Comment:
Går det an å fjerne toList() her?

.map { } returnerer allerede en liste, så det blir redudant 
```suggestion
        manuellBrevmottakerRepository.insertAll(manuelleBrevmottakere)
```